nilearn/input_data/multi_nifti_masker.py

```diff
@@ -185,7 +185,7 @@ def fit(self, imgs=None, y=None):
                             target_shape=self.target_shape,
                             n_jobs=self.n_jobs,
                             memory=self.memory,
-                            verbose=(self.verbose - 1),
+                            verbose=max(0, self.verbose - 1),
```

Comment:
There are other occurences of this verbose - 1 elsewhere in the code. Would you mind fixing them while you are at it?

```
~/dev/nilearn $ git grep -n -P 'verbose\s*-\s*1'
nilearn/group_sparse_covariance.py:548:                verbose=self.verbose - 1, debug=False)
nilearn/input_data/multi_nifti_masker.py:188:                            verbose=(self.verbose - 1),
nilearn/input_data/nifti_masker.py:168:                verbose=(self.verbose - 1),
```